wiringPiD/drcNetCmd.h

```diff
@@ -34,11 +34,19 @@
 #define	DRCN_DIGITAL_READ8	8
 #define	DRCN_ANALOG_READ	9
 
-
+// Multiple definition for comDat fix
```

Comment:
Why not just delete this rather than comment it out?

Reply:
Yeah, I agree.  I'm very confused by Github's pull request / review interface at this point, but I'm going to try to get this thing merged with the extra definition deleted.  Apologies if things get more mangled along the way.  :)